core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -627,6 +627,48 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
     assert(exc.getCause() != null)
     stream.close()
   }
+
+  test("support barrier sync under local mode") {
```

Comment:
we don't have `barrier sync` yet, do we need to test it now?

Reply:
Updated the title, but I think the test body is still needed to make sure barrier execution mode works with SparkContext.